api/src/shared/load-env-file.js

```diff
@@ -0,0 +1,10 @@
+import { existsSync } from 'node:fs';
+import * as url from 'node:url';
+
+const envFilePath = url.fileURLToPath(new URL('../../.env', import.meta.url));
+
+export function loadEnvFile() {
```

Comment:
Est-ce que cette fonction ne gagnerait pas être renommée en `loadEnvFileIfExists` ? Est-ce que cela ne serait pas plus cohérent avec l’utilisation de l’option `--env-file-if-exists` ?
```suggestion
export function loadEnvFileIfExists() {
```